style/content-block.css

```diff
@@ -25,6 +25,19 @@ details {
   position: relative;
 }
 
+.c-relatedBox {
+  position: relative;
+  border: 1px solid #3892a7;
```

Comment:
Er dette en farge som vi eventuelt har i en variabel?

Reply:
Ikke som jeg kan se. Tok den fra Invision. Kan opprette den som en variabel?

Reply:
Ok, er egentlig supert om vi kan bruke farger som variabler, evt. hvis du finner en lignende?

Jeg ser for eksempel at vi har dette i vår css: 
```css
--article-color: #5090a4;
```
som er delvis lignende? Kan godt renavne denne variablen hvis du bruker den 😄